### DISCUSSION TITLE
dependabot configured, but not work

### DISCUSSION THREAD

**Discussion by njfamirm** [community/community#74682] [Actions] [answered]
### Select Topic Area

Bug

### Body

Hi

I'm [configured](https://github.com/Alwatr/containers/blob/next/.github/dependabot.yml) dependabot in my repo.
but when I'm going to the [dependabot log page](https://github.com/Alwatr/containers/network/updates), see this message
```
Dependabot version updates aren't configured yet
Dependabot creates pull requests to keep your dependencies up-to-date.

Learn more
```
my config is absolutely correct, and I'm even checked dipandabat in the settings
But I still haven't been able to fix it!

this issue appear in all project in our org, all new project, all transformed repo, even when last project in my profile work

thank you

**Comment by alimd** [comment]
I have to explain this problem further

It doesn't matter whether we create a new project or move the previous project (in which the bot is active and working)
Under no circumstances do we dare to activate this bar in our new org!

For example for this new repo:
https://github.com/Alwatr/containers

Our `.github/dependabot.yml` is https://github.com/Alwatr/containers/blob/next/.github/dependabot.yml

Project setting:
![Screenshot 2023-11-08 at 9 17 12 PM](https://github.com/community/community/assets/697913/b2a74658-3877-43ce-8e5f-c2cde95b35be)

Organization setting:
![Screenshot 2023-11-08 at 9 19 42 PM](https://github.com/community/community/assets/697913/02420b24-8cfc-48f8-878d-fc3b4b5485ad)

Many new commit, disable, re-enable config, delete dependabot.yml and copy from github docs, Reading all related issues in the world... We did everything we could think of, but it didn't work 😭 

Always see this message for all repos!
![Screenshot 2023-11-08 at 9 34 46 PM](https://github.com/community/community/assets/697913/a5d701fa-5700-46a1-a1d6-898f528f490d)

**Comment by njfamirm** [comment, marked as answer]
this issue resolved with manually trigger in github support
